src/main/java/study/SplitUtil.java

```diff
@@ -0,0 +1,27 @@
+package study;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class SplitUtil {
+
+    public static final String SPLIT_DELIMITER = "[,:]";
+
+    private SplitUtil() {
+    }
+
+    public static List<Integer> splitExpression(String expression) {
+        Matcher matcher = Pattern.compile("//(.)\\n(.*)").matcher(expression);
+
+        if (matcher.find()) {
+            String customDelimiter = matcher.group(1);
+            return Arrays.stream(matcher.group(2).split(customDelimiter)).map(Integer::parseInt).collect(Collectors.toList());
+        }
+
+        return Arrays.stream(expression.split(SPLIT_DELIMITER)).map(Integer::parseInt).collect(Collectors.toList());
```

Comment:
```suggestion
        Matcher matcher = Pattern.compile("//(.)\\n(.*)")
                                 .matcher(expression);

        if (matcher.find()) {
            String customDelimiter = matcher.group(1);
            return Arrays.stream(matcher.group(2).split(customDelimiter))
                         .map(Integer::parseInt)
                         .collect(Collectors.toList());
        }

        return Arrays.stream(expression.split(SPLIT_DELIMITER))
                     .map(Integer::parseInt)
                     .collect(Collectors.toList());
```
한 줄에 dot(`.`)은 최대한 적게 유지해주세요!

Reply:
그리고 이렇게 놓고보니 메소드 하나에 10줄이 넘어가고 있습니다. 어떻게 하면 더 줄일 수 있을지 고민해보면 좋을 것 같아요~

Reply:
SplitUtil.splitExpression()에서 split만 하도록 변경했습니다!